app/assets/javascripts/hyku_addons/select_toggleable.js

```diff
@@ -0,0 +1,33 @@
+class SelectToggleable {
+  parentSelector = ".js-select-toggle"
+  addSelector = ".js-select-toggle-add"
+  removeSelector = ".js-select-toggle-remove"
+  changeSelector = ".js-select-toggle-change"
+
+  constructor(){
+    this.registerEvents()
+  }
+
+  registerEvents(){
+    $("body").on("click", this.addSelector, this.addSection.bind(this))
+    $("body").on("click", this.removeSelector, this.removeSection.bind(this))
+    $("body").on("change", this.changeSelector, this.changeSection.bind(this))
+  }
+
+  addSection(event){
+    event.preventDefault()
+  }
+
+  removeSection(event){
+    event.preventDefault()
+  }
```

Comment:
Hi Chris. Yea my hope is that because of the similarities in the specific section JavaScript files, we can reduce them all down to a few actions here and some callbacks. That's the hope anyway.